photo-app/app/assets/javascripts/credit_card_form.js

```diff
@@ -0,0 +1,61 @@
+const flashMessages = document.getElementById("flash-messages");
+
+const showError = function (message) {
+  if(!(flashMessages).length){
```

Comment:
Did you test this?

Reply:
havent yet, just push. Need to include jqeury to test this. After i test i will request for review again